test/jest/Netscript/Bladeburner.test.ts

```diff
@@ -14,6 +15,18 @@ const skill = new Skill({
 });
 
 describe("Test calculateMaxUpgradeCount", function () {
+  it.failing("Bug: Returns 1 when SP is too small and current level is too high", () => {
```

Comment:
Please name these with the expected value of how they should work, not the current buggy state. That way when they are fixed they can just be flipped to `it(`

Reply:
You know I'm not actually sure what the correct behavior is (at least the one's we **wish** to fix according to the discussion in the issue).

Reply:
If you don't know what the correct behavior is, then it's probably not the right time to be writing tests. XD

Reply:
I don't know how to write the test you suggested (the inverse case). The test I wrote covers the bug, not the fix.

Reply:
I think you misunderstood what I was asking for. You don't need to be able to fix the bug, or even write a fully fleshed out "regular" test in a situation like this. However, you ought to be able to at least describe in words, for the test description, how the code "ought" to behave, if you think it actually has a bug. Even just inverting what you have ("Should not return 1 at high levels with small SP") would be sufficient, and explains the construction of your test.